fmriprep/workflows/anatomical.py

```diff
@@ -81,35 +81,38 @@ def t1w_preprocessing(name='t1w_preprocessing', settings=None):
     if settings['freesurfer']:
         nthreads = settings['nthreads']
 
-        def detect_inputs(t1w_list, default_flags=''):
+        def detect_inputs(t1w_list, t2w_list=[]):
+            import os
+            from nipype.interfaces.base import isdefined
             from nipype.utils.filemanip import filename_to_list
+            from nipype.interfaces.traits_extension import Undefined
             import nibabel as nib
             t1w_list = filename_to_list(t1w_list)
+            t2w_list = filename_to_list(t2w_list) if isdefined(t2w_list) else []
             t1w_ref = nib.load(t1w_list[0])
-            hires = max(t1w_ref.header.get_zooms()) < 1
+            hires = round(max(t1w_ref.header.get_zooms()), 1) < 1
```

Comment:
Woops. Lost the thread in the rebase. Recap:

@oesteban suggested we set an explicit threshold, rather than round.

```Python
hires = np.all(np.array(t1w_ref.header.get_zooms()) - 1 < -tol)
```

Where `tol` here `= 0.05`.

If we're going to do that, I'd prefer to stick with the `max` formulation, and keep constants on the right-hand-side, e.g.:

```Python
hires = max(t1w_ref.header.get_zooms()) < 1 - 0.05
```

I'll do whatever the consensus is, but I think the `round(max(zooms), 1) < 1` and `max(zooms) < 1 - tol` are the most readable options. Opinions, @chrisfilo?